src/Entities/Player.cpp

```diff
@@ -339,6 +314,79 @@ void cPlayer::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 
 
+void cPlayer::TickFreezeCode(bool a_MyChunkIsSent)
+{
+	// This function is ticked by the player's client handle. This ensures it always ticks, even if the player
+	// is standing in an unloaded chunk, unlike cPlayer::Tick. We need this because the freeze handling code must
```

Comment:
About if it would help, and if it would be be a good thing to have it as
soon as possible? Yes :). A big, blinking, singing and dancing YES.
Connectivity/lag spike/void issues have been a thorn in everyone's side for
far too long, and some players thought it pointless to even try and connect
after a while, because sometimes they fell from the sky, sometimes they
suffocate down below, or fell through the world, and when they were just
stuck in a way they had to practically take the spawn point apart by
desperately digging themselves out. :) It's been a bit stressy as of late.

On Mon, Apr 4, 2016 at 1:23 PM, LogicParrot <notifications@github.com>
wrote:

> In src/Entities/Player.cpp
> <https://github.com/cuberite/cuberite/pull/3097#discussion_r58361376>:
>
> > @@ -339,6 +314,79 @@ void cPlayer::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
> >
> >
> >
> > +void cPlayer::TickFreezeCode(bool a_MyChunkIsSent)
> > +{
> > +	// This function is ticked by the player's client handle. This ensures it always ticks, even if the player
> > +	// is standing in an unloaded chunk, unlike cPlayer::Tick. We need this because the freeze handling code must
>
> Isn't that fine though? Do we want a floating pack of skeletons to shoot
> at a slowly sinking player while a chunk is loading?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cuberite/cuberite/pull/3097/files/dd0ce3287ffddf3fd0dd5d7af1ebda60bd871c85#r58361376>
>